pkg/kudoctl/util/repo/package.go

```diff
@@ -0,0 +1,326 @@
+package repo
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/bundle"
+	"github.com/pkg/errors"
+	"io"
+	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	"regexp"
+	"sigs.k8s.io/yaml"
+	"strings"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	frameworkV0FileName = "-framework.yaml"
+	versionV0FileName   = "-frameworkversion.yaml"
+	instanceV0FileName  = "-instance.yaml"
+
+	frameworkV1FileName = "framework.yaml"
+	templateV1FileNameRegex = "templates/.*.yaml"
+	paramsV1FileName = "params.yaml"
+)
+
+const apiVersion = "kudo.k8s.io/v1alpha1"
+
+type InstallCRDs struct {
```

Comment:
InstallCRDs is kind of a weird name for a type in my mind for some reason. I know it's technically right and I can't think of a better one, just wanted to observe that.

Reply:
maybe 
```suggestion
type BundleCRDs struct {
``` 
or
```suggestion
type BundleV1CRDs struct {
``` 
or
```suggestion
type V1Alpha1CRDs struct {
``` 
?

Reply:
Also dislike the `InstallCRDs` name. It's CRDs now, but I'm sure we'll have more file types at some point.

Reply:
I went with `PackageCRDs` - we can always change that :)

Reply:
Just don't forget all the parameters and methods still named `*InstallCRDs` ;)